src/config/firebase.ts

```diff
@@ -0,0 +1,16 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyCTm784xoUyYForelcga-P_30C0tyurhcM",
+  authDomain: "rock-paper-scissors-c74f9.firebaseapp.com",
+  projectId: "rock-paper-scissors-c74f9",
+  storageBucket: "rock-paper-scissors-c74f9.appspot.com",
+  messagingSenderId: "386384777615",
+  appId: "1:386384777615:web:4392ea93e8cd4c3892bbfa",
+};
```

Comment:
It is **very unprotected** to save API keys in a simple file and push them to a public repo. Rework it with `.env` file